public/PROPOSAL/particle/ParticleDef.h

```diff
@@ -332,29 +332,29 @@ PROPOSAL_MAKE_HASHABLE(PROPOSAL::ParticleDef, t.mass, t.lifetime, t.charge)
 namespace PROPOSAL {
 static std::map<const int, const ParticleDef&> Id_Particle_Map
```

Comment:
wouln't `const ParticleType` as type for the map be better than all those casts to int?